packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -36,7 +36,7 @@ module.exports = (resolve, rootDir) => {
     testURL: 'http://localhost',
     transform: {
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
-      '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
+      '^.+\\.tsx?$': resolve('node_modules/ts-jest/preprocessor.js'),
```

Comment:
I'm not sure what exactly should be used here.
My first attempt was to use
```
resolve('<rootDir>/node_modules/ts-jest/preprocessor.js')
```
but that failed on the first run - figure out that `resolve` does a simple path joining here. Using the line above the first test part (L127 in e2e-simple.sh) passes, but the second one (with the generated test project, L264) fails.
In an ejected project, this should be 
```
'^.+\\.tsx?$': '<rootDir>/node_modules/ts-jest/preprocessor.js'
```
Using this directly also fails on the first test run, since `<rootDir>` references a different project root not containing ts-jest.

It seems that it'd be required to use different paths in both situations, but I'm still trying to figure out how to set this up properly.